File: Linear/Static/Array/03_Deleting/Given_Position/Built-In/Deleting.js
import readline from 'readline'

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})

let arr = [10, 20, 30, 40]

console.log("Array before deletion")
console.log(arr.join(" "))

rl.question("\nEnter a position: ", (ans) => {
    const pos = parseInt(ans)

    // Delete the element at the given position.
    arr.splice(pos - 1, 1)

    console.log("Array after deletion")
    console.log(arr.join(" "))
    
    rl.close()
})

File: Linear/Static/Array/03_Deleting/Beginning/Built-In/Deleting.py
arr = [10, 20, 30, 40]

print("Array before deletion")
for i in range(len(arr)):
    print(arr[i], end=" ")

# Remove the first element.
del arr[0]

print("\nArray after deletion")
for i in range(len(arr)):
    print(arr[i], end=" ")